tcf_website/api/filters.py

```diff
@@ -1,12 +1,20 @@
 """"Custom DRF pagination classes"""
-from django_filters import FilterSet, NumberFilter
-from ..models import Instructor
+from django_filters import FilterSet, BooleanFilter, NumberFilter
+from ..models import Instructor, Semester
 
 
 class InstructorFilter(FilterSet):
     """Filter for Instructor"""
     section__course = NumberFilter(field_name='section__course', distinct=True)
+    recent5years = BooleanFilter(method='filter_recent5years', distinct=True)

```

Comment:
Definitely think calling the param `recent` makes more sense from a legibility perspective — just add a comment somewhere in the code / documentation so that it's clear that we know what "recent" means.

Reply:
I'm gonna make this change (and fix pylint) and merge it in since it's blocking review form rn